src/components/form/form.style.js

```diff
@@ -16,14 +19,49 @@ export const StyledAdditionalFormAction = styled.div`
       display: inline-block;
     `}
 
-    ${StyledButton} {
+    ${applyButtonLeftMargin}
+
```

Comment:
white space.

Reply:
Yeah I wanted to leave it there so it's more readable